Cargo.lock

```diff
@@ -1,7 +1,5 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
```

Comment:
Can we install the walkdir crate without updating the crates in this PR? (we can do it in a separate PR)